src/ActorMultiVertex.cpp

```diff
@@ -194,12 +205,21 @@ void ActorMultiVertex::AddVertices( int Add )
 {
 	int size = AMV_DestTweenState().vertices.size();
 	size += Add;
-	for( std::size_t i = 0; i < AMV_Tweens.size(); ++i )
+
+	for (std::size_t i = 0; i < AMV_Tweens.size(); ++i)
```

Comment:
Since this code is duplicated, I would add a small private helper function that just reserves space in these three vector to a specific size.